e2e/regressions/build.go

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
```

Comment:
I'm not sure this is the best place code organizational wise for this, but I'm happy to have this here for now. Thanks!